### DISCUSSION TITLE
My application cannot be reviewed until you fix the following

### DISCUSSION THREAD

**Discussion by pattysung1** [community/community#88622] [GitHub Education]
### Hi everyone,

After I submit, I'm now have this problem:

"Your application cannot be reviewed until you fix the following:
Your request has expired, please try again"

and my jpg file is 547KB, so I think that is fit in the size.

Still can't figure out why.

![image](https://github.com/community/community/assets/113825171/0d35f182-116b-458c-813c-4d0fc978421c)

**Comment by Nekokir** [comment]
same here
